ecl/resource_ecl_provider_connectivity_tenant_connection_request_v2.go

```diff
@@ -0,0 +1,274 @@
+package ecl
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+
+	"github.com/nttcom/eclcloud"
+	"github.com/nttcom/eclcloud/ecl/provider_connectivity/v2/tenant_connection_requests"
+	"github.com/nttcom/eclcloud/ecl/sss/v1/approval_requests"
+)
+
+func resourceProviderConnectivityTenantConnectionRequestV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceProviderConnectivityTenantConnectionRequestV2Create,
+		Read:   resourceProviderConnectivityTenantConnectionRequestV2Read,
+		Update: resourceProviderConnectivityTenantConnectionRequestV2Update,
+		Delete: resourceProviderConnectivityTenantConnectionRequestV2Delete,
+		Schema: map[string]*schema.Schema{
+			"tenant_id_other": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"network_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"tags": &schema.Schema{
+				Type:     schema.TypeMap,
+				Optional: true,
+			},
+			"status": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"keystone_user_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"tenant_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"name_other": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
It is not request parameter.
https://ecl.ntt.com/en/documents/api-references/provider-connectivity/rsts/APIList/source/v2.0/TC/create_tenant_connection_request.html#parameters
```suggestion
				Computed: true,
```